composer.json

```diff
@@ -8,7 +8,8 @@
         "php-http/message": "^1.0",
         "php-http/client-common": "^1.1",
         "php-http/discovery": "^1.0",
-        "webmozart/assert": "^1.2"
+        "webmozart/assert": "^1.2",
+        "egulias/email-validator": "^2.1"
```

Comment:
This should be removed.

Reply:
Yep, agree. I’ll proceed with the steps that we discussed the last time we met and discussed this case. I’ll keep you posted for a final review.